vscode/src/client.ts

```diff
@@ -171,6 +183,7 @@ export default class Client extends LanguageClient implements ClientInterface {
     createTestItems: (response: CodeLens[]) => void,
     workspaceFolder: vscode.WorkspaceFolder,
     outputChannel: WorkspaceChannel,
+    mainWorkspace = false,
```

Comment:
Can we prefix this with `is`, or make the name indicate better that it's a boolean?
Asking because in other places, we have variables like `firstWorkspace` or `workspace` that actually present `workspaceFolder` objects. I think it'll help readability if we can make a clearer distinction between this variable and those.